### ISSUE TITLE
Indentation after attribute on type declaration

### ISSUE THREAD

**Issue by leviroth** [OCamlPro/ocp-indent#257]
Currently indents as:

```ocaml
module M = struct
  type a = A of b [@@deriving compare]
and b = B of a
end
```

Should indent as:

```ocaml
module M = struct
  type a = A of b [@@deriving compare]
  and b = B of a
end
```

I'm not sure if this is related to other issues people have had with attributes.

Version: 1.6.0

**Comment by AltGr** [comment]
Confirmed on master. Thanks for reporting!

**Comment by nojb** [comment]
This issue is fixed in 1.7.0, can be closed.
